api/nodes/serializers.py

```diff
@@ -99,13 +106,49 @@ class NodePointersSerializer(JSONAPISerializer):
 
     id = ser.CharField(read_only=True, source='_id')
     node_id = ser.CharField(source='node._id')
-    title = ser.CharField(source='node.title')
+    title = ser.CharField(read_only=True, source='node.title')
 
     class Meta:
         type_ = 'pointers'
 
     links = LinksField({
-        'html': 'absolute_url',
+        'html': 'get_absolute_url',
+    })
+
+    def get_absolute_url(self, obj):
+        pointer_node = Node.load(obj.node._id)
+        return pointer_node.absolute_url
+
+    def create(self, validated_data):
+        request = self.context['request']
+        user = request.user
+        auth = Auth(user)
+        node = self.context['view'].get_node()
```

Comment:
Also, you probably don't have to pass both view and request. You can get `request` from the View object.